sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyCurveName.cs

```diff
@@ -13,9 +13,9 @@ namespace Azure.Security.KeyVault.Keys
     /// </summary>
     public struct KeyCurveName : IEquatable<KeyCurveName>
     {
-        internal readonly Oid Oid;
-        internal readonly int KeySize;
-        internal readonly int KeyParameterSize;
+        internal readonly Oid _oid;
```

Comment:
I don't think you should do this for non-private fields, including `internal` which have public-like scope for its assembly and friend assemblies. Only privates should have the underscore and it should never be exposed to code outside the class.

Reply:
I guess that makes sense, but we are trying to follow corefx rules closely. This is what a section of their doc states 

>3. We use `_camelCase` for internal and private fields and use `readonly` where possible. Prefix internal and private instance fields with `_`, static fields with `s_` and thread static fields with `t_`. When used on static fields, `readonly` should come after `static` (e.g. `static readonly` not `readonly static`).  Public fields should be used sparingly and should use PascalCasing with no prefix when used.

You can find the whole style doc [here](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)
@pakrym what do you think?

Reply:
Talked offline and while it doesn't look great in code, it's good to be consistent. I can always have a get-accessor for the field if I feel strongly about it. It will likely be inlined.